src/home/homed-manager.c

```diff
@@ -512,6 +512,8 @@ static int search_quota(uid_t uid, const char *exclude_quota_path) {
                 if (r < 0) {
                         if (ERRNO_IS_NOT_SUPPORTED(r))
                                 log_debug_errno(r, "No UID quota support on %s, ignoring.", where);
+                        else if (ERRNO_IS_PRIVILEGE(r))
+                                log_debug_errno(r, "UID quota support for %s prohibited, ignoring.", where);
```

Comment:
I think something like this would be a tiny bit better, but it doens't matter much ("Quota support prohibited" doesn't sound well):
```C
if (ERRNO_IS_NOT_SUPPORTED(r) || ERRNO_IS_PRIVILEGE(r))
  log_debug_errno(r, "No UID quota support on %s, ignoring: %m", where);
```